### PR TITLE
SAK-48397 Samigo: Uncaught TypeError: (...).sortable is not a function

### PR DESCRIPTION
**Pull request by kunaljaykam**
https://sakaiproject.atlassian.net/browse/SAK-48397

### CODE REVIEW

File: library/src/webapp/bundled-js/sakai.morpheus.reorderer.js [hunk 1]
```diff
@@ -106,15 +106,19 @@ $PBJQ(document).ready(function(){
...
+            items: '.reorder-element:not(.notsortable)',
+            start: function( event, ui ) {
+            preserveStatus(ui);
+            },
+            update: function( event, ui ) {
+            registerChange(event, ui);
+            },
+        });
+    } else {
+        return null;
```
Comment (adrianfish) [inline library/src/webapp/bundled-js/sakai.morpheus.reorderer.js:120]: What happens when you return null from here?

Comment (kunaljaykam) [reply library/src/webapp/bundled-js/sakai.morpheus.reorderer.js:120]: no error, because it won't execute it unless that id is present.

Comment (kunaljaykam) [reply library/src/webapp/bundled-js/sakai.morpheus.reorderer.js:120]: If the statement makes sure not to run .sortable() if the id is not present in the markup, therefore no error.

**Review by adrianfish — APPROVED** [review]
GTG. Thanks Kunal.

**Pull request merged**